tests/libwnbd_tests/test_disk_actions.cpp

```diff
@@ -457,3 +458,154 @@ TEST(TestUnmap, TestHardUnmapFallback) {
 TEST(TestUnmap, TestHardUnmap) {
     TestUnmap(false, 0, 0, true);
 }
+
+TEST(TestPersistentReservations, TestPersistentReserveIn) {
+    auto InstanceName = GetNewInstanceName();
+
+    MockWnbdDaemon WnbdDaemon(
+        InstanceName,
+        DefaultBlockCount,
+        DefaultBlockSize,
+        false,
+        false,
+        false,
+        false,
```

Comment:
MockWnbdDaemon arguments are getting out of hand, we'll have to merge the custom flags in a single argument. We'll do it in a separate patch.